LTI-FJLTT/LTI-FJLTT.md

```
@@ -0,0 +1,385 @@
+El objetivo central es reducir la carga operativa de los recruiters, mejorar la colaboración con los hiring managers y automatizar tareas repetitivas mediante IA responsable.  
+
+El MVP se centrará en parsing y matching IA, flujo visual de pipeline, colaboración en tiempo real y cumplimiento RGPD.  
+La diferenciación se logra con una UX moderna, automatizaciones simples y explicabilidad IA nativa.  
+El modelo SaaS por vacantes activas reduce fricción comercial y facilita escalado por cliente.  
+Azure se selecciona por su integración con ecosistemas empresariales y servicios gestionados de seguridad (Entra ID, Key Vault, Defender).
+
+```mermaid
+graph TD
+A[**Problema**] -->|1| B[**Segmentos de Clientes**]
+A -->|2| C[**Propuesta de Valor Única**]
+C -->|3| D[**Solución**]
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 187

(MD010, no-hard-tabs)